src/modelcompiler.cpp

```diff
@@ -2,32 +2,23 @@
 // Licensed under the terms of the GPL v3. See licenses/GPL-3.txt
 
 #include "buildopts.h"
-#include "core/Log.h"
```

Comment:
Did these changes get included in this review by mistake? I cannot see how they're related to the atmosphere rendering